### DISCUSSION TITLE
How to design my profile on github?

### DISCUSSION THREAD

**Discussion by uchiha11111** [community/community#132496] [Discussions]
### Select Topic Area

Question

### Body

I have seen many people designing their github profile. How to do that?

**Comment by devmuse7** [comment]
You have to first create a New Repository named after your GitHub Username. 
Enable the box of "Add README file"
Create Repository and ensure it's public
Once the Repository is created, click on README.md file. Click on Edit button (pencil icon) at top right corner.
Edit your README file. You can add images. links etc. and then commit your changes.
Now, Navigate to your GitHub profile and can see the content of README.md displayed on your profile page.
You can also use some websites to edit your README.md file.

**Reply by LiteBrite82** [reply]
Hi @Itachii27 

We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
